container/lxd/initialisation_linux.go

```diff
@@ -79,21 +77,36 @@ func (ci *containerInitialiser) Initialise() error {
 	if err := ensureDependencies(ci.lxdSnapChannel, localSeries); err != nil {
 		return errors.Trace(err)
 	}
-	err = ci.configureLxdBridge()
-	if err != nil {
-		return errors.Trace(err)
-	}
 	proxies := proxy.DetectProxies()
 	err = ci.configureLxdProxies(proxies, ci.isRunningLocally, ci.newLocalServer)
 	if err != nil {
 		return errors.Trace(err)
 	}
 
-	output, err := ci.getExecCommand(
-		"lxd",
-		"init",
-		"--auto",
-	).CombinedOutput()
+	lxdInitCfg := `config: {}
+networks: []
+storage_pools:
+- config:
+    size: 5GiB
+  description: ""
+  name: default
+  driver: zfs
+profiles:
+- config: {}
+  description: ""
+  devices:
+    root:
+      path: /
+      pool: default
+      type: disk
+  name: default
```

Comment:
I'm not sure about `default` here.

Reply:
I removed it from the `--preseed` yaml, but it gets added to the default profile (because lxd has not been init'd yet) anyway